primitives/io/src/lib.rs

```diff
@@ -1704,6 +1845,7 @@ pub type SubstrateHostFunctions = (
 	wasm_tracing::HostFunctions,
 	offchain::HostFunctions,
 	crypto::HostFunctions,
+	elliptic_curves::HostFunctions,
```

Comment:
```suggestion
```

No need to add this by default right now.

Reply:
Moved it to `primitives/crypto-ec-utils` crate.